shell/platform/darwin/macos/framework/Source/FlutterEngineTest.mm

```diff
@@ -535,7 +535,56 @@ @implementation MockableFlutterEngine
   // TODO(gw280): add support for screenshot tests in this test harness
 
   [engine shutDownEngine];
-}  // namespace flutter::testing
+}
+
+TEST_F(FlutterEngineTest, CompositorIgnoresUnknownView) {
+  FlutterEngine* engine = GetFlutterEngine();
+  auto original_init = engine.embedderAPI.Initialize;
+  ::FlutterCompositor compositor;
```

Comment:
Are the `::` needed?

```suggestion
  FlutterCompositor compositor;
```

Reply:
Yes, because we have two `FlutterCompositor` classes. The `::FlutterCompositor` is the embedder API one, while the `flutter::FlutterCompositor` is the macOS one.